packages/ui/react-ui-theme/src/config/tokens.ts

```diff
@@ -162,8 +162,6 @@ const physicalSeries = {
   ...huePalettes,
 };
 
-export type HuePalette = keyof typeof huePalettes;
```

Comment:
I believe this type was being used somewhere — did removing this not break the build?

Reply:
Ah nevermind, I see it’s exported from another file.

Reply:
refactored so that we can avoid exporting lots of defs to a lib that should really only be part of the plugin. recommend splitting up packages like this into plugin and lib directories.

Reply:
> a lib that should really only be part of the plugin

Do you mean to say the hue palettes should not be defined in `react-ui-theme`? We already have `react-ui-theme` and `plugin-theme`, and I’m amenable to moving definitions between them as needed, though I think there are good reasons hue palettes are defined in this package and not elsewhere.